src/main/java/org/openbase/bco/bcozy/util/Groups.java

```diff
@@ -63,5 +84,33 @@ public String toString(UnitConfigType.UnitConfig object) {
         };
     }
 
+    public static void removeGroup(UnitConfigType.UnitConfig group) throws InterruptedException {
+        try {
+            Registries.getUserRegistry().removeAuthorizationGroupConfig(group);
+        } catch (CouldNotPerformException ex) {
+            ExceptionPrinter.printHistory(ex, LOGGER);
```

Comment:
Again, how do you inform the user if this fails?

Reply:
This is part of the goal I set myself for this week (as I told you last monday) and therefore not implemented in this pullrequest.